consul/client.go

```diff
@@ -12,14 +12,51 @@ import (
 	"time"
 
 	"github.com/hashicorp/consul/consul/structs"
+	"github.com/hashicorp/consul/lib"
 	"github.com/hashicorp/serf/coordinate"
 	"github.com/hashicorp/serf/serf"
 )
 
 const (
```

Comment:
Ok - I'd start up an issue with a checklist of places to hit and capture this there so we keep track of it and remember to add tests.